insights/parsers/vdsm_conf.py

```diff
@@ -1,8 +1,8 @@
 """
-VDSMConfIni - file ``/etc/vdsm/vdsm.conf``
-==========================================
+VDSMConfIni & VDSMLoggerConf
+============================
 
-The VDSM configuration file ``/etc/vdsm/vdsm.conf`` is in the standard 'ini'
+``VDSMConfIni`` parsers configuration file ``/etc/vdsm/vdsm.conf``
```

Comment:
Thanks for the updating. 
For multiple parsers in one python file, how do you think about listing a summary in the file header and move the `sample` and `examples` into each class, just like what it does in odbc.py or ntp_source.py?

See: http://insights-core.readthedocs.io/en/latest/shared_parsers_catalog/odbc.html

Reply:
@xiangce 

I planned to do exactly similar, but as you see the test `test_documentation()` will not work as it works on a module(AFAIK)